webextensions/manifest-keys.json

```diff
@@ -2074,6 +2074,176 @@
               "version_added": false
             }
           }
+        },
+        "color": {
```

Comment:
I think this is "colors" (plural). (Like in https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/theme )